helm-charts/infisical/templates/backend-deployment.yaml

```diff
@@ -70,3 +61,17 @@ spec:
     - protocol: TCP
       port: 4000
       targetPort: 4000 # container port 
+
+---
+
+{{ if not $backend.kubeSecretRef }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ $backend.kubeSecretRef | default "infisical-backend" }}
```

Comment:
Shouldn't the name default to `include "infisical.backend.fullname" .`? Otherwise multiple helm install without customization will have colliding names.